configator/create_gatorgradle_yml.py

```diff
@@ -0,0 +1,50 @@
+""""Program to automatically generate gatorgrader.yml files in a config directory"""
+
+# import necessary libraries
+import os
+
+print("Let's work on getting your configuration files generated!")
+
+def create_gatorgrader():
+    """creating directory that can have files
+    that can be read and written to.
+    """
+    # creating directory called config
+    directory = "config"
+
+    os.mkdir(directory)
```

Comment:
This will fail if the `config` directory already exists. You should handle this case by not creating that directory (maybe the user has other config files).